scipy/spatial/distance.py

```diff
@@ -1373,6 +1373,12 @@ def dice(u, v, w=None):
     dice : double
         The Dice dissimilarity between 1-D arrays `u` and `v`.
 
+    Notes
+    -----
+    This function computes the Dice dissimilarity index. To compute the 
+   Dice similiarity index, convert one to the other with similarity = 
+   1 - dissimilarity.
```

Comment:
You will have to align these two lines (1379 and 1380) also by indenting them by one whitespace.

Reply:
Thanks @tirthasheshpatel. I have updated this and all checks are now passed.

Reply:
You're welcome @leockl.

Reply:
Do I need to close this PR or do anything now?

Reply:
No. Just wait for the someone with write access to the repository review and approve your pr. Once approved, it may be merged in the master branch of the repository and you can then delete your branch.

Reply:
Ok great! Many thanks once again, appreciate it!